### DISCUSSION TITLE
中文论文的标题设置字号无效果

### DISCUSSION THREAD

**Discussion by YiweiSu** [note286/xduts#155] [Q&A] [answered]
背景：专硕中文论文，已下载最新版xduts，texlive更新至最新版，除此问题其他使用都很流畅，已深入阅读xduts.pdf。

按照学校学位论文字号的要求（二级标题小三号），我参考xduts.pdf里的5.15节设置了style/sec-zihao=-3，但是没有效果，又测试了一下chap-zihao仍然没有更新标题字号的效果，同时找了[issue](https://github.com/note286/xduts/issues/131)，是类似问题但我的无法更改标题字号，烦请大佬帮忙解答一下，感激不尽！

**Comment by note286** [comment]
研究生的标题字号是规定死的，xduts实现的也没有问题， https://github.com/note286/xduts/issues/131 是本科生的。建议你再检查一下是否看错了要求。

**Reply by YiweiSu** [reply]
非常感谢回复！

资料参考：[西安电子科技大学研究生学位论文模板（2015版）-申请专业学位硕士研究生使用-2022.11](https://gr.xidian.edu.cn/system/_content/download.jsp?urltype=news.DownloadAttachUrl&owner=1281831001&wbfileid=6597845)
我看的这个的第2.10节，2022.11修订版的，里面提到了：
> 二级标题不缩进，字体为宋体加粗，字号为小三号，行距为固定值20磅，段落间距为段前18磅，段后12磅；

**Reply by note286** [reply]
你为啥会觉得xduts中二级标题不是小三呢？

**Reply by YiweiSu** [reply]
xduts.pdf不是说了么：
设置章节标题字号。当论文语言为中文时，默认值分别为 3、4、4、4、4 和 4，分别对应chap、sec、subsec、subsubsec、para、subpara。

**Reply by note286** [reply, marked as answer]
看来你并没有深入阅读手册，只有本科生支持设置，我在前面的回答中已经提及了，这是本科生的。

![image](https://github.com/note286/xduts/assets/16207292/24fa7121-4a6b-43f0-a540-99571eb6331e)

**Reply by YiweiSu** [reply]
没注意到，谢谢大佬~
